bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RowMerger.java

```diff
@@ -364,8 +364,8 @@ private FlatRow buildRow() {
 
   private RowMergerState state = RowMergerState.NewRow;
   private ByteString lastCompletedRowKey = null;
-  private RowInProgress rowInProgress;
-  private boolean complete;
+  private RowInProgress rowInProgress = null;
+  private boolean complete = false;
   private int rowCountInLastMessage = -1;
```

Comment:
This is probably out of scope for this PR, so feel free to ignore this question, but I was wondering why this initialized to -1 instead of 0?

Reply:
It probably should be null and not either -1 or 0.  I'll update that.